### PR TITLE
feat: pid flow in wallet

### PR DESCRIPTION
**Pull request by janrtvld**
I tried to re-use some screens but i've copied over a lot of the logic because it's hard to understand code and some things just work a bit different if you already have a wallet. So to not make the logic in the onboarding even more unreadable i just did it like this.

TODOs
- [x] Test on Android

### CODE REVIEW

File: packages/ui/src/components/IdCard.tsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ export function IdCard({ icon, issuerImage, backgroundImage, userName, onPress,
             {onPress && <HeroIcons.ArrowRight color="$black" />}
           </XStack>
         </XStack>
-        <Stack zIndex="$-1" position="absolute" top={0} left={0} right={0} bottom={0}>
+        <Stack zIndex="$-1" position="absolute" bg="#F1F2F0" top={0} left={0} right={0} bottom={0}>
```
Comment (TimoGlastra) [inline packages/ui/src/components/IdCard.tsx:106]: shouldn't this color come from the theme?

File: apps/easypid/src/features/pid/PidReviewRequestSlide.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  const { onNext } = useWizard()
+  const display = usePidDisplay()
+
+  const [isLoading, setIsLoading] = useState(false)
+
+  const onContinue = () => {
+    if (isLoading) return
+    setIsLoading(true)
+
+    // Do logic
```
Comment (TimoGlastra) [inline apps/easypid/src/features/pid/PidReviewRequestSlide.tsx:23]: what do you mean with do logic? shouldn't we do the logic here? isn't it implemented yet?

**Review by TimoGlastra — CHANGES_REQUESTED** [review]
To me this duplication doesn't make sense, and the reason being the complexity makes less sense. I think we just made it 2x as complex by dupilcating the code. Everything now needs to be changed in two places. 

I think the correct way would be to create a better abstraction. So we can at least reuse the views  and logic, but still have control over what is different.

At least we should have the screens as separate components

File: apps/easypid/src/features/pid/PidReviewRequestSlide.tsx (hunk 2 above)
Comment (janrtvld) [reply apps/easypid/src/features/pid/PidReviewRequestSlide.tsx]: This was placeholder, removed it.

**Review by TimoGlastra — APPROVED** [review]

**Pull request merged**